internal/page/app_data.go

```diff
@@ -82,3 +81,11 @@ func (d AppData) EncodeQuery() string {
 
 	return query
 }
+
+func (d AppData) OrganisationName() string {
+	if d.SupporterData == nil {
+		return ""
+	}
+
+	return d.SupporterData.OrganisationName
+}
```

Comment:
This was to simplify some logic in templates but I'm not sure if this negates the boundary created by having `SupporterData`?

Reply:
Decided against this in the end